CHANGELOG.md

```diff
@@ -2,16 +2,22 @@
 
 ## [Unreleased]
 
+### Added
+
+- More CRD documentation ([#354]).
+
 ### Changed
 
+- `operator-rs` `0.56.1` -> `0.57.0` ([#354]).
 - Increase resource defaults ([#352]).
 
 ### Fixed
 
 - BREAKING: Fixed various issues in the CRD structure. `clusterConfig.credentialsSecret` is now mandatory ([#353]).
 
-[#352]: https://github.com/stackabletech/airflow-operator/pull/352
```

Comment:
Nit: elsewhere in the changelog these PR numbers are in descending order, I think